tests/unit/Rules/SuperSetTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Singwai Chan <c.singwai@gmail.com>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @group  rule
+ * @covers \Respect\Validation\Rules\SuperSet
+ * @covers \Respect\Validation\Exceptions\SuperSetException
+ */
+class SuperSetTest extends TestCase
```

Comment:
Could you extend [RuleTestCase](https://github.com/Respect/Validation/blob/1.1/tests/library/Rules/RuleTestCase.php) instead? Check this out: https://github.com/Respect/Validation/blob/1.1/CONTRIBUTING.md#creating-unit-tests